src/scenes/App/index.js

```diff
@@ -28,6 +29,7 @@ const App = injectIntl(observer(({ classes, intl, match: { url }, store, store:
     {!store.loading && (
       <Fragment>
         <div className={classes.container}>
+          <NavBar />
```

Comment:
no props need to be passed to it?

Reply:
don't think we need any props. all the page route is working